bookshelf/5-logging/src/main/java/com/example/getstarted/daos/DatastoreDao.java

```diff
@@ -118,15 +117,15 @@ public void deleteBook(Long bookId) {
     if (startCursorString != null && !startCursorString.equals("")) {
       startCursor = Cursor.fromUrlSafe(startCursorString);    // Where we left off
     }
-    Query<Entity> query = Query.entityQueryBuilder()          // Build the Query
-        .kind("Book6")                                         // We only care about Books
-        .limit(10)                                            // Only show 10 at a time
-        .startCursor(startCursor)                             // Where we left off
-        .orderBy(OrderBy.asc(Book.TITLE))                     // Use default Index "title"
+    Query<Entity> query = Query.newEntityQueryBuilder()       // Build the Query
+        .setKind("Book5")                                     // We only care about Books
```

Comment:
Yep or some variable that comes from a POM or somewhere else -- this whole idea has issues.